sources/pods.go

```diff
@@ -83,6 +83,12 @@ func (self *realPodsApi) parsePod(podNodePair *podNodePair) *api.Pod {
 		localContainer := api.Container{}
 		localContainer.Name = container.Name
 		localContainer.Image = container.Image
+		if cpu, ok := container.Resources.Requests[kapi.ResourceCPU]; ok {
+			localContainer.Spec.CpuRequest = cpu.MilliValue()
```

Comment:
We will eventually need both request and limits I guess. Why not follow kube api semantics and make requests and limits a map[resourceType]Quantity?